runtime-modules/content/src/tests/mod.rs

```diff
@@ -5,6 +5,6 @@ mod curators;
 mod fixtures;
 mod merkle;
 mod mock;
-mod nft;
+//mod nft;
```

Comment:
Please, add TODO comment here.

Reply:
fixed in 93de823c63787575d87f47916ccd33f7530469be